### DISCUSSION TITLE
How do you open the Configuration if the tray icon is not visible?

### DISCUSSION THREAD

**Discussion by bunk3m** [flameshot-org/flameshot#2895] [Q&A]
So I unselected the "Show tray icon" to see what happens.  And so the top menu icon for Flameshot went away as expected.

Now I want to Show the tray icon again but I can't get to the Configuration menu.  Flameshot is running in the background but I can't get to the Configuration menu.  Yes, I can get to the color selector but not any of the preferences.

I've searched the documentation and I assume there is a way to do this since there is a preference "on" & "off".  But I can't find it.

How do I open the Configuration if the tray icon is not visible?

Thanks in advance.

Intel Mac - Monterey 12.5.1
Flameshot
Flameshot v12.1.0 (96c2c82e)
Compiled with Qt 5.15.5
darwin: 21.6.0
osx: 12.5

**Comment by borgmanJeremy** [comment]
You can run flameshot config from the cli

**Comment by bunk3m** [comment]
Thanks @borgmanJeremy I didn't realize that was possible.
Where would I find the instructions on how to do this?  It isn't in the docs that I could see.
Since I've installed the app to /Applications and not installed from Macports or Brew, I don't have a command line version running anywhere.

**Reply by borgmanJeremy** [reply]
It’s documented in the readme and website. You can still use the cli if you call it from the full path /Applications/…/flameshot 

on mobile so I don’t have the full path handy

**Comment by bunk3m** [comment]
Thanks @borgmanJeremy 

Found the man page and used the following to open the configuration settings.

`/Applications/flameshot.app/Contents/MacOS/flameshot config`
